crates/sui-move/src/build.rs

```diff
@@ -67,7 +67,13 @@ impl Build {
             },
         )?;
         if dump_bytecode_as_base64 {
-            println!("{}", json!(pkg.get_package_base64(with_unpublished_deps)))
+            println!(
+                "{}",
+                json!({
+                    "modules": pkg.get_package_base64(with_unpublished_deps),
+                    "dependencies": json!(pkg.get_package_dependencies_base_64())
```

Comment:
Note that you'll also need to do the same validation logic that @rvantonder added to the `publish` flow (check for no `unpublished` deps, unless `--with-unpublished-deps` is passed, etc.